core/storage/base_model/gae_models.py

```diff
@@ -520,7 +526,8 @@ def get(cls, entity_id, strict=True, version=None):
             version: int. Version we want to get. Default is None.
 
         Returns:
-            VersionModel.
+            VersionedModel. If version is None get the newest version, otherwise
```

Comment:
If version is None, get the newest version of the model. Otherwise, get the specified version.